src/app/layout/Layout.tsx

```diff
@@ -1,25 +1,34 @@
+import React from "react";
+import { Outlet } from "react-router-dom";
+import { Sidebar } from "../sidebar";
+import css from "./Layout.module.css";
 
-import React from 'react'
-import css from './Layout.module.css'
-
-interface LayoutProps {
-    children: React.ReactNode
-    sidebar: React.ReactNode
+interface BaseLayoutProps {
+    children: React.ReactNode;
+    sidebar: React.ReactNode;
 }
 
-
-export const Layout = ({ children, sidebar }: LayoutProps) => {
-
+export const BaseLayout = ({ children, sidebar }: BaseLayoutProps) => {
     return (
-            <div className={css.wrapper}>
+        <div className={css.wrapper}>
             <aside className={css.sidebar}>{sidebar}</aside>
-                <div
-                    className={css.main}
-                >
-                    {children}
-                </div>
-            </div>
+            <div className={css.main}>{children}</div>
+        </div>
     );
-}
+};
+
+export const BaseLayoutWithSidebar = ({
```

Comment:
Previously this was defined in the `Routes` component, as to not import `Sidebar` in this layout component, so it didn't depend on another component. However, I think this simplifies things since the error-routes also need to render the layout with the sidebar, but not the outlet in the normal `Layout` below.